Sources/GRPCCore/Call/Client/Internal/ClientRPCExecutor+OneShotExecutor.swift

```diff
@@ -98,11 +98,13 @@ extension ClientRPCExecutor.OneShotExecutor {
   ) async -> Result<R, any Error> {
     return await withTaskGroup(of: Void.self, returning: Result<R, any Error>.self) { group in
       do {
-        return try await self.transport.withStream(descriptor: method, options: options) { stream in
+        return try await self.transport.withStream(descriptor: method, options: options) {
+          stream,
+          context in
```

Comment:
nit: swift-format did a bad job here

```suggestion
        return try await self.transport.withStream(
          descriptor: method, 
          options: options
        ) { stream, context in
```